pages/_next-prefetcher.js

```diff
@@ -0,0 +1,108 @@
+/* global self */
```

Comment:
@nkzawa I tried this now and It'll add ~28K with webpack but without it it's just 4K. Do we still need to proceed with webpack or just use the code written with ES5.